src/handlers/transformation/build-event-name.ts

```diff
@@ -0,0 +1,34 @@
+import { CsvRow } from "./transform-row";
+
+interface Rules {
+  "Minimum Level Of Assurance": string;
+  "Billable Status": string;
+  "Event Name": string;
+}
+
+export interface EventNameRules {
+  [key: string]: Rules[];
+}
+
+export async function buildEventName(
+  eventNameRules: EventNameRules,
+  idpEntityId: string,
+  row: CsvRow
+): Promise<string> {
+  const minLevelOfAssurance = row["Minimum Level Of Assurance"];
+  const billableStatus = row["Billable Status"];
+
+  const rules = eventNameRules[idpEntityId];
+
+  if (rules !== undefined) {
+    for (const rule of rules) {
```

Comment:
Try not to use `for`-loops so extensively, try to `rules.find()` instead here.

Reply:
Thanks for the tip. Will make the change.